### PR TITLE
docs: styling page updates

### PR DESCRIPTION
**Pull request by zanettin**
additions requested on #2125

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by zanettin** [comment]
@manucorporat you were too qwik with the merge 😆  here are the requested additions.

**Comment by manucorporat** [comment]
sorry! i didn't wanted to block the PR because of what we could add extra! thank you so much for taking the time to opening another one! you are amazing!

File: packages/docs/src/routes/docs/components/styles/index.mdx [hunk 1]
```diff
@@ -53,7 +53,7 @@ Component styles allow Qwik to lazy load the style information for the component
 
 ```tsx
 import {useStyles$} from '@builder.io/qwik';
-import styles from './code-block.css?inline';
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/components/styles/index.mdx:56]: i saw you reverted this! it was correct

Comment (zanettin) [reply packages/docs/src/routes/docs/components/styles/index.mdx:56]: omg! nice catch! wasn't on purpose 🙈  reverted 👍

**Pull request merged**